spec/fork_break_spec.rb

```diff
@@ -86,6 +86,20 @@ def counter_after_synced_execution(counter_path, with_lock)
     end
   end
 
+  it 'ignores breakpoints when running outside a ForkBreak process' do
+    class Foo
+      include ForkBreak::Breakpoints
+
+      def bar
+        breakpoints << :test
+        'baz'
+      end
+    end
+
+    expect { Foo.new.bar }.to_not raise_error
+    expect(Foo.new.bar).to eq('baz')
+  end
```

Comment:
If possible I would do the assertion at actual process level, something
like `expect(current_process).to_not be_fork_break`

On Thu, Nov 5, 2015 at 1:12 PM, Pedro Carriço <notifications@github.com>
wrote:

> In spec/fork_break_spec.rb
> <https://github.com/forkbreak/fork_break/pull/14#discussion_r44004931>:
>
> > @@ -86,6 +86,20 @@ def counter_after_synced_execution(counter_path, with_lock)
> >      end
> >    end
> >
> > +  it 'ignores breakpoints when running outside a ForkBreak process' do
> > +    class Foo
> > +      include ForkBreak::Breakpoints
> > +
> > +      def bar
> > +        breakpoints << :test
> > +        'baz'
> > +      end
> > +    end
> > +
> > +    expect { Foo.new.bar }.to_not raise_error
> > +    expect(Foo.new.bar).to eq('baz')
> > +  end
>
> do you think that:
>
> expect(Foo.new.breakpoints).to be_kind_of(ForkBreak::NullBreakpointSetter)
>
> would suffice?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/forkbreak/fork_break/pull/14/files#r44004931>.
>